internal/exec/atmos.go

```diff
@@ -151,9 +166,13 @@ func ExecuteAtmosCmd() error {
 		configAndStacksInfo.ProcessFunctions = true
 		err = ExecuteTerraform(configAndStacksInfo)
 		if err != nil {
+			log.Debug(fmt.Sprintf("Error executing Terraform command: %v", atmosConfig.Settings.Telemetry.Enabled))
+
+			telemetry.CaptureCmdFailureString(fmt.Sprintf(telemetryCommandFormat, selectedCommand))
```

Comment:
This makes me worried that we could log sensitive information.

Reply:
Done